src/guacenc/ffmpeg-compat.h

```diff
@@ -78,5 +88,81 @@
  */
 int guacenc_avcodec_encode_video(guacenc_video* video, AVFrame* frame);
 
+/**
+ * Creates and sets up the AVCodecContext for the appropriate version
+ * of libavformat installed. The AVCodecContext will be built, but
+ * the AVStream will also be affected by having its time_base field
+ * set to the value passed into this function.
+ *
+ * @param stream
+ *     The open AVStream
```

Comment:
As with JavaDoc, the formatting of these Doxygen comments requires that the description of parameters, return values, etc. all end with periods. This is correct for the other new code - some sneaky creature just went through this one and ate all the periods. ;)

Reply:
Ping @sreid8 - any chance you could finish this up?  Looks like it's about ready to merge so we can close out for next release.